AirportTicketBooking/Enums/ManagerOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace AirportTicketBooking.Enums;
+
+public enum ManagerOptions
+{
+    GetUser,
+    GetAllUsers,
+    UpdateUser,
+    DeleteUser,
+    GetBooking,
+    GetAllBookings,
+    UpdateBooking,
+    DeleteBooking,
+    GetFlight,
+    GetAllFlights,
+    UpdateFlights,
+    DeleteFlights,
+    Exit
```

Comment:
Let's give them explicit constant values
```suggestion
    GetUser = 1,
    GetAllUsers = 2,
    UpdateUser = 3,
    DeleteUser = 4,
    GetBooking = 5,
    GetAllBookings = 6,
    UpdateBooking = 7,
    DeleteBooking = 8,
    GetFlight = 9,
    GetAllFlights = 10,
    UpdateFlights = 11,
    DeleteFlights = 12,
    Exit = 0
```